batch_import/camio_hooks.py

```diff
@@ -61,6 +78,39 @@ def set_hook_data(data_dict):
         CAMIO_SERVER_URL = "https://test.camio.com"
     Log.debug("setting camio_hooks data as:\n%r", CAMIO_PARAMS)
 
+def get_account_info():
+    """
+    this function takes an auth token and gathers both the device ID of their
+    Camio Box and then uses that device ID to query for the current local IP
+    address. This way the user doesn't have to supply those items to use manually
+    """
+    access_token = get_access_token()
+    device_id = get_device_id(fail=False)
+    ip_address = CAMIO_PARAMS.get('ip_address')
+    if not device_id:
+        url = CAMIO_SERVER_URL + CAMIO_DEVICES_ENDPOINT
+        ret = network_request('get', url)
+        if not ret:
+            fail("unable to obtain account info (device_id and Box IP address")
+        devices = ret.json()
+        if not devices or len(devices) < 1:
+            fail('no Camio Box devices found on your account, have you registered your Box yet?')
+        device_id = devices[0]['device_id']
+        CAMIO_PARAMS['device_id'] = device_id
+    if not ip_address:
+        url = CAMIO_SERVER_URL + CAMIO_STATE_ENDPOINT + "?device_id=%s" % device_id
+        ret = network_request('get', url)
+        device = ret.json()
+        network_config = device.get('state').get('network_configuration_actual')
+        if not network_config:
+            fail("unable to obtain IP address of Camio box")
```

Comment:
Use proper capitalization "Camio Box" and differentiate between the two errors like:
```python
fail("Camio Box hasn't yet reported its IP address; please retry in 2 minutes.")